lib/registration.pm

```diff
@@ -158,7 +158,7 @@ sub register_addons {
                 send_key_until_needlematch "scc-code-field-$addon", 'tab';
             }
             else {
-                assert_and_click "scc-code-field-$addon";
+                assert_and_click "scc-code-field-$addon", 60;
```

Comment:
It should be: `assert_and_click "scc-code-field-$addon", 'left', 60;`

see also: [poo#35197](https://progress.opensuse.org/issues/35197)

Reply:
updated by: https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/4886